src/components/board/TagsTabSidebard.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+	<div>
+		<ul class="labels">
+			<li v-for="label in labels" :key="label.id">
+				<template v-if="editingLabelId === label.id">
+					<input v-model="editingLabel.title"><input v-model="editingLabel.color">
+					<button class="icon-checkmark" @click="updateLabel(label)" :disabled="!editLabelObjValidated" 
+					v-tooltip="{content: 'title and color must be provided', 
+					show: !editLabelObjValidated, trigger: 'manual' }" />
+
+					<button v-tooltip="t('deck', 'Cancel')" class="icon-close" @click="editingLabelId = null" />
+				</template>
+				<template v-else>
+					<span :style="{ backgroundColor: `#${label.color}`, color: `#white` }" class="label-title">
```

Comment:
White shouldn't be hardcoded. We should have a proper mixin to calculate the text color for a given background color.